samples/java-protobuf-customer-registry-kafka-quickstart/docker-compose.yml

```diff
@@ -13,31 +13,38 @@ services:
         -Dkalix.proxy.eventing.support=kafka
       USER_FUNCTION_HOST: ${USER_FUNCTION_HOST:-host.docker.internal}
       USER_FUNCTION_PORT: ${USER_FUNCTION_PORT:-8080}
-      BROKER_CONFIG_FILE: /conf/my-local.kafka.properties
-    volumes:
-      - .:/conf
+      # configuring kafka broker used for eventing
+      BROKER_SERVERS: kafka:29092
+      # Alternatively:
+      # - for easy local/dev mode - plaintext, for non dev-mode TLS, use:
+      #BROKER_AUTH_MECHANISM: "PLAIN"
+      #BROKER_USERNAME: [your-username]
+      #BROKER_PASS: [your-pass]
+      # - for SCRAM-SHA-256 or SCRAM-SHA-512, use:
+      #BROKER_AUTH_MECHANISM: "SCRAM-SHA-256" #OR SCRAM-SHA-512
+      #BROKER_CA_CERT: [your-cert-location]
```

Comment:
Note that for scram they would actually have to script adding users to zookeeper interacting with the zookeeper node and a whole mess, so maybe skip mentioning that